docker-hub/scim.md

```diff
@@ -2,175 +2,63 @@
-    > When you create an Enterprise Application you can configure SCIM with
-    > Security Assertion Markup Language (SAML).
-
-3. Type **Docker** for application name, select **non-gallery**, and **Create**.
-4. In your Docker application, navigate to **Provisioning**, and select **Get Started**.
-5. Select **Automatic** for the provisioning mode and enter your SCIM credentials.
-
-    > **Note**
-    >
-    > You can access and copy your SCIM URL (Tenant URL) and API Token (Secret
-    > Token) in Docker Hub.
-
-6. Select **Test Connection** to enable and authorize the provisioning.
-7. In **Mappings**, select **Provision Azure Active Directory Groups** to disable Groups and **Save**.
-
-    > **Note**
-    >
-    > You must turn off group sync to avoid having your configuration
-    > quarantined.
-8. Select **Provision Azure Active Directory Users**, in **Attribute Mappings**, and keep the **userName**, **Active**, **givenName**, **familyName**. Delete the other attributes listed.
-
-    ![attribute-mapping](images/attribute-mapping.png){:width="700px"}
-
-9. Select **Start Provisioning** to begin the full synchronization.
+## Set up SCIM
 
-     > **Note**
-     >
-     > It can take up to 30 minutes to begin provisioning. You can also
-     > provision on demand provisioning with one user that's already assigned
-     > to the application.
+You must make sure you have [configured SSO](../single-sign-on/index.md) before you enable SCIM. Enforcing SSO is not required.
 
-### Onelogin
+### Step one: Enable SCIM in Docker Hub
 
-1. In Onelogin, navigate to **Applications** > **Applications** > **Add app**.
-2. In the search field, enter **SCIM Provisioner with SAML (SCIM v2 Core)** and select the item in the results.
-3. Enter **Docker Hub** as the display name and **Save**.
-4. Navigate to the left navigation, and select **Configuration**.
-5. In a separate tab, navigate to **Docker Hub** > **Settings** > **Security** > **SSO** > **SAML**.
-6. Copy the following fields from Docker Hub, in to OneLogin:
+1. In the **Single Sign-On Connections** table, select the **Actions** icon and **Setup SCIM**.
 
-    * Entity ID: SAML Audience URL
-    * ACS URL:  SAML Consumer URL
-    * SCIM Base URL: SCIM Base URL
-    * Custom Headers:
+2. Copy the **SCIM Base URL** and **API Token** and paste the values into your IdP.
 
-    ```console
-     Content-Type: application/scim+json
-     User-Agent: OneLogin SCIM
-    ```
-    * SCIM Bearer Token: SCIM Bearer Token
-    * SCIM JSON Template:
+### Step two: Enable SCIM in your IdP
 
-    ```console
-    {
-    "schemas": [
-        "urn:ietf:params:scim:schemas:core:2.0:User"
-    ],
-    "userName": "{$parameters.scimusername}",
-    "name": {
-        "givenName": "{$user.firstname}",
-        "familyName": "{$user.lastname}"
-    },
-    "emails": [
-        {
-            "value": "{$user.email}",
-            "primary": true
-        }
-    ]
-}
-    ```
+Follow the instructions provided by your IdP:
```

Comment:
@ivan-californias Would you mind checking if I've provided the right links for users? 🙏 

cc @MercedesSteele